lib/transformer.dart

```diff
@@ -5,6 +5,7 @@ library initialize.transformer;
 
 import 'dart:async';
 import 'dart:collection' show Queue;
+import 'package:analyzer/src/dart/element/element.dart';
```

Comment:
Actually it was not! It is necessary because of the usage of `VariableElementImpl.evaluationResult`.